lib/public/IUserSession.php

```diff
@@ -80,4 +80,17 @@ public function getUser();
 	 * @since 8.0.0
 	 */
 	public function isLoggedIn();
+
+	/**
+	 * get getImpersonatingUserID
+	 * @return string | null
+	 * @since 17.0.0
+	 */
+	public function getImpersonatingUserID();
```

Comment:
```suggestion
	public function getImpersonatingUserID(): ?string;
```
(or is that only with the next php version?)